<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
    <item>
     <widget class="QLineEdit" name="urlEdit">
      <property name="placeholderText">
       <string>https://</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetBrowser" native="true">
      <layout class="QHBoxLayout" name="browserLayout"/>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="logEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>120</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="actionRefresh"/>
  </widget>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
